sys/conf/ldscript.mips

```diff
@@ -260,7 +271,7 @@ SECTIONS
       .bss section disappears because there are no input sections.  */
    . = ALIGN(64 / 8);
   }
-  . = ALIGN(64 / 8);
+  . = ALIGN(0x100000);
```

Comment:
Do we want to be padding like this on hybrid and vanilla kernels?

Reply:
If it hurts image size too much I can pass the file through cpp, not sure if it is worth it for 1MiB padding at most.

Reply:
I _think_ it should also be possible to use ALIGN(_BSS_ALIGN) and then pass `-Wl,-defsym=_BSS_ALIGN=0x10000` in the purecap case and `-Wl,-defsym=_BSS_ALIGN=8` otherwise.

Reply:
Just checked and this works with ld.lld.